### DISCUSSION TITLE
诺奖得主侃大山 - Miao Yu | 于淼

### DISCUSSION THREAD

**Discussion by giscus** [yufree/yufree.cn#429] [General]
# 诺奖得主侃大山 - Miao Yu | 于淼

https://yufree.cn/cn/2023/08/07/nobel/

**Comment by NetZissou** [comment]
好文章。我推断有大可能性存在Agency专门签约不同背景的但是共同点都是有fame，或有与众不同的，难以复刻的经历的harmless character，安排他们出席讲演，上电视节目，录播客，做group training session. 给科学家群体派Richard Roberts，给Corporate America office job群体派ex-criminals, ex-mafia boss.

**Reply by yufree** [reply]
感觉美国演讲市场挺火的，做博后时系里还花钱请人过来讲如何做报告啥的，这类名人演讲应该也有相应产业链，而且都是老客户，一年或两年来一次，吃定百十个不差钱的客户就可以养家糊口了，每周两个报告外带跟中介八二分的话估计手里十几个名人资源就能玩转养活公司。
